lib/methods/get-orders/response-builder.js

```diff
@@ -0,0 +1,24 @@
+const buildGetOrderssResponse = response => {
+  const getItem = item => ({
+    id: Number(item.id.$value),
+    status: Number(item.status.$value),
+    customerReference: String(item.customerReference.$value)
+  });
+
+  let orders =
```

Comment:
I think a variable named `responseItem` would be more appropriate. The value assigned to it is the raw items (if they exist) from the response.